### DISCUSSION TITLE
Format for Pasted Markdown

### DISCUSSION THREAD

**Discussion by sim2kid** [outline/outline#6266] [Ideas]
Outline already supports markdown formatting if you type it out, but if you paste Markdown into the editor, the editor needs to do more work to format it.

As such, it is difficult to move over sections of markdown documentation into Outline. Doing this can get tedious as I must go through each section and reformat it in the editor.

In addition, there's currently no support for markdown tables, let alone pasted tables.

Here is an example of something I may try to paste:
``` Markdown
The `SubtitleController` class is a MonoBehaviour implemented in Unity, and it's responsible for controlling the visual aspect of subtitles in a Unity game.

| Name | Data Type | Description |
| --- | --- | --- |
| `Show()` | `void` | Makes the subtitle visible by changing the 'Visable' flag in the Animator and updating the 'visible' property. |
```

Here is the current vs what is expected:
![image](https://github.com/outline/outline/assets/13566418/f0ccc424-a678-423e-8090-6faf444dbf6e)

The current Actual result should only be true if formatting is disabled using **Shift + Ctrl + V**, which is standard on other platforms.

**Comment by tommoor** [comment]
It depends where you paste it from to some extent, if you paste this content in plain text it converts just fine (tested it) – the problem is when there is HTML on the clipboard and we can't detect as Markdown then the app will default to using the HTML which will often represent the plaintext version of the content.

**Reply by tommoor** [reply]
Just pushed an update so that table headings will also be used in Markdown detection

**Reply by sim2kid** [reply]
I double-checked and the typical markdown does indeed work when pasting. I may have had an issue in my environment before preventing me from accomplishing this.

I appreciate you adding the Table Headings check for Markdown.
